Doc/library/sqlite3.rst

```diff
@@ -919,11 +897,17 @@ Connection objects
 
       .. versionadded:: 3.7
 
-
    .. method:: getlimit(category, /)
 
-      Get a connection runtime limit. *category* is the limit category to be
-      queried.
+      Get a connection runtime limit.
+
+      :param int category:
+         The `SQLite limit category`_ to be queried.
+
+      :rtype: int
+
+      :raises ProgrammingError:
+         If *category* is not recognised by the underlying SQLite library.
```

Comment:
```suggestion
      Get a connection runtime limit.

      :param int category:
         The `SQLite limit category`_ to be queried.
      :rtype: int
      :raises ProgrammingError:
         If *category* is not recognised by the underlying SQLite library.
```
What do you think about removing the empty lines?
It saves some space and groups all these related info in a single block.

Reply:
For functions with few parameters, it may work well, but for functions like `sqlite3.connect` and `sqlite3.Connection.backup`, I think it makes the raw file more messy; I would not do that for those. For consistency, I'd prefer to use the same style everywhere, so I'm -1 to this :)